import/import.go

```diff
@@ -145,9 +145,14 @@ func doImport(f io.Reader, size int, addTags string,
 // write or something, overwrite it.
 func writeFile(path string, buf []byte) (err error) {
 	const flags = os.O_WRONLY | os.O_CREATE | os.O_TRUNC
+openFile:
 	f, err := os.OpenFile(path, flags, 0660)
 	if err != nil {
-		return
+		if os.MkdirAll(path, 0660) != nil {
```

Comment:
`path` is the file path. You will need to strip the last segment.
`~/.hydron/images/aa/aa43464554544535.jpg` -> `~/.hydron/images/aa/`

Reply:
Okay, is there anything else that needs to be done?
I'm kind of walking in the dark.